packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/GoogleMapFactory.java

```diff
@@ -47,11 +47,11 @@ public PlatformView create(@NonNull Context context, int id, @Nullable Object ar
     builder.setInitialHeatmaps(params.getInitialHeatmaps());
     builder.setInitialTileOverlays(params.getInitialTileOverlays());
 
-    final String cloudMapId = mapConfig.getCloudMapId();
-    if (cloudMapId != null) {
-      builder.setMapId(cloudMapId);
+    final String mapId = mapConfig.getMapId();
```

Comment:
For my own education can you share why we would want to use mapId instead of cloudMapId and why the change is safe (wont cause a regression in existing apps)

Reply:
I've added a comment about renaming it [here.](https://github.com/flutter/packages/pull/7882#discussion_r1893813468). It's safe because `cloudMapId` can still be used by existing clients, it's marked as deprecated and uses `mapId` under the hood